src/views/ScenarioManager/ScenarioManager.js

```diff
@@ -124,27 +125,36 @@ const ScenarioManager = (props) => {
     findScenarioById(WORKSPACE_ID, scenarioId);
   };
 
+  const canUserDeleteScenario = (scenario) => {
```

Comment:
if you create a UserPermissionsHooks you mutualize by create  a function useCallback that check a particular right.
Like isUserHavePermission( CONST PERMISSION )